src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/Primitives/Selector.cs

```diff
@@ -1857,7 +1857,7 @@ private void ItemSetIsSelected(ItemInfo info, bool value)
         }
 
         /// <summary>
-        /// Returns false if FrameworkElement representing this item has Selector.SelectableProperty set to false.  True otherwise.
+        /// Returns false if FrameworkElement is Separator.  True otherwise.
```

Comment:
```
/// <summary>
/// Returns false if FrameworkElement representing this item is not selectable. True otherwise. If the FrameworkElement is Separator or null, we return False.
/// </summary>
```

How does this sound ?

Reply:
Thank you @dipeshmst 

And I fix the comment